src/custom_source_buffers/text/html/html_text_source_buffer.ts

```diff
@@ -73,16 +78,34 @@ function generateClock(videoElement : HTMLMediaElement) : Observable<boolean> {
 
 /**
  * @param {Element} element
- * @param {Element|null} [child]
+ * @param {Element} child
  */
-function safelyRemoveChild(element : Element, child : Element|null) {
-  if (child != null) {
-    try {
-      element.removeChild(child);
-    } catch (e) {
-      log.warn("HTSB: Can't remove text track: not in the element.");
-    }
+function safelyRemoveChild(element : Element, child : Element) {
+  try {
+    element.removeChild(child);
+  } catch (e) {
+    log.warn("HTSB: Can't remove text track: not in the element.");
```

Comment:
It disturbs me to declare the input as `e`. Could we name it `_` ?

Reply:
As we does not us it.

Reply:
done

Reply:
yes good